plugins/toolbox-search/src/block_searcher.ts

```diff
@@ -72,20 +87,33 @@ export class BlockSearcher {
         })
         .values(),
     ];
+    const resultState = result.map((item) => JSON.parse(item));
+    return resultState;
```

Comment:
You should be able to set this up to just `return` from the top. But I can't put a suggestion over this whole function b/c github :P
```suggestion
    ].map(JSON.parse);
```